### PR TITLE
Notify inviter when invitation is accepted

### PR DESCRIPTION
**Pull request by HSalmon**
screen shot 2016-02-18 at 1 04 06 pm

### CODE REVIEW

**Comment by rdbartlett** [comment]
:heart_eyes_cat:

**Comment by robguthrie** [comment]
in the string "accepted your invitation to join Loomio" is Loomio the group
name? or is it just the string Loomio? I think it should be the group name.

**Comment by mdshearer** [comment]
Yee har!! And yeah, group name would be swell.

**Comment by getresults** [comment]
Having clickable URLs to both the person's loomio profile and the specific loomio group would be awesome if you haven't included this already. :)

**Comment by HSalmon** [comment]
User name links to user profile, group name links to group:

screen shot 2016-02-18 at 2 07 44 pm

**Comment by HSalmon** [comment]
@gdpelican, @robguthrie, would love yr eyes when you've got a moment or two?

File: spec/models/events/invitation_accepted_spec.rb [hunk 1]
```diff
@@ -1,16 +0,0 @@
-require 'rails_helper'
```
Comment (gdpelican) [inline spec/models/events/invitation_accepted_spec.rb]: Did these tests fail?

File: app/models/notification.rb [hunk 2]
```diff
@@ -10,9 +26,9 @@ class Notification < ActiveRecord::Base
   delegate :eventable, :to => :event
 
   scope :unviewed, -> { where(viewed: false) }
+  scope :beta_notifications, -> { joins(:event).where(:events => {:kind => BETA_NOTIFICATIONS}) }
```
Comment (gdpelican) [inline app/models/notification.rb]: Brilliant! I think we want to stay away from the hash rocket though (as a matter of style)

```
joins(:event).where(events: { kind: BETA_NOTIFICATIONS })
```

**Comment by gdpelican** [comment]
This looks pretty fantastic to me. An e2e would be fantastic. Here's my thought:

- Send invitation
- visit `development/accept_last_invitation`, which accepts the invite on behalf of the invitee, like this:
```
def accept_last_invitation
  InvitationService.redeem(Invitation.last, Invitation.last.invitee)
  redirect_to(test_group)
end
```
- Click notifications dropdown, I should have a notification about it
- Click the notification, I should be on the memberships page
- On that page, I should see the invited person's name or handle or something

**Comment by HSalmon** [comment]
I'm having a lot of trouble implementing your suggestions for an e2e @gdpelican, would you be up for pairing on it?

**Pull request merged**